apps/s_client.c

```diff
@@ -1424,6 +1424,13 @@ int s_client_main(int argc, char **argv)
     if (argc != 0)
         goto opthelp;
 
+#ifndef OPENSSL_NO_NEXTPROTONEG
+    if (min_version == TLS1_3_VERSION
+        && max_version == TLS1_3_VERSION && next_proto_neg_in != NULL) {
+        BIO_printf(bio_err, "Cannot supply NPN with TLSv1.3\n");
+        goto end;
```

Comment:
Since it's not described in `-help` that NPN is not allowed in TLSv1.3, if goto opthelp, not really helps : )